backend/vcdat/app.py

```diff
@@ -76,7 +76,11 @@ def plot_template():
     tmpl = request.get_json()
     t = templ_from_json(tmpl)
     canvas = vcs.init(bg=True)
-    g = vcs.getboxfill()
+    g = vcs.createboxfill()
+    g.xmtics1 = {.5 * i: "" for i in range(1,20,2)}
+    g.xmtics2 = g.xmtics1
+    g.ymtics1 = g.xmtics1
+    g.ymtics2 = g.xmtics1
```

Comment:
I just hardcoded xticlabels and yticlabels, that solved the problem. You're in the clear, @doutriaux1